index4_1.js

```diff
@@ -0,0 +1,29 @@
+/* получить результат {"65432445": 'The Chamber', "675465": 'Fracture', "70111470": 'Die Hard', "654356453": 'Bad Boys'}*/
+
+
+const videos = [
+  {
+    id: 65432445,
+    title: "The Chamber",
+  },
+  {
+    id: 675465,
+    title: "Fracture",
+  },
+  {
+    id: 70111470,
+    title: "Die Hard",
+  },
+  {
+    id: 654356453,
+    title: "Bad Boys",
+  },
+];
+
+const createObjectWithValue = (videos) => {
+  return videos.reduce((result, { id, title }) => {
+    result[`"${id}"`] = title;
```

Comment:
result[id] is enough. Keys in in objects are always strings cause before creating a key toString() method happens automatically